process/hyperBlockProcessor.go

```diff
@@ -1,29 +1,30 @@
 package process
 
 import (
-	"encoding/hex"
-
 	"github.com/ElrondNetwork/covalent-indexer-go/hyperBlock"
-	"github.com/ElrondNetwork/covalent-indexer-go/schema"
+	"github.com/ElrondNetwork/covalent-indexer-go/schemaV2"
 )
 
 type hyperBlockProcessor struct {
+	transactionProcessor TransactionHandler
 }
 
 // NewHyperBlockProcessor will create a new instance of an hyper block processor
-func NewHyperBlockProcessor() *hyperBlockProcessor {
-	return &hyperBlockProcessor{}
+func NewHyperBlockProcessor(transactionHandler TransactionHandler) *hyperBlockProcessor {
+	return &hyperBlockProcessor{
+		transactionProcessor: transactionHandler,
+	}
 }
 
 // Process will process current hyper block and convert it to an avro schema block result
-func (hbp *hyperBlockProcessor) Process(hyperBlock *hyperBlock.HyperBlock) (*schema.BlockResult, error) {
-	block := schema.NewBlockResult()
+func (hbp *hyperBlockProcessor) Process(hyperBlock *hyperBlock.HyperBlock) (*schemaV2.HyperBlock, error) {
+	block := schemaV2.NewHyperBlock()
```

Comment:
I would 
```suggestion
	hyperBlockV2 := schemaV2.NewHyperBlock()
```
and propagate it down the function.

Reply:
No need to use `v2` in variable name, since I will delete "`v1`" it after the whole refactor is done;
Changed the variable name to `avroHyperBlock`, however